### PR TITLE
Change package injection

### PR DESCRIPTION
**Pull request by benbc**

### CODE REVIEW

**Comment by benbc** [comment]
@spacecowboy This is my proposed change to the way we do Docker pipeline stuff. Can you take a look, please?

**Comment by spacecowboy** [comment]
Trying it locally and it doesn't work for me.

```
$ cat devenv.local
# -*- mode: shell-script -*-

NEO4J_EDITION=community
NEO4J_VERSION=3.0.2
```

```
$ source devenv.local
```

```
$ make
Makefile:13: *** NEO4J_EDITION is not set.  Stop.
zsh: exit 2
```

```
$ echo $NEO4J_EDITION
community
```

And before you suspect `zsh`:

```
$ bash
[jonas@armitage docker-neo4j]$ . devenv.local
[jonas@armitage docker-neo4j]$ make
Makefile:13: *** NEO4J_EDITION is not set.  Stop.
[jonas@armitage docker-neo4j]$ echo $NEO4J_EDITION
community
```

**Comment by spacecowboy** [comment]
```
$ make --version
GNU Make 4.2.1
Built for x86_64-pc-linux-gnu
Copyright (C) 1988-2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

**Comment by spacecowboy** [comment]
Also, what the heck is `record-env`?

```
$ which record-env
record-env not found
```

**Comment by benbc** [comment]
@spacecowboy Sorry. You have to source `devenv`. That includes `devenv.local`.

**Comment by benbc** [comment]
`record-env` is in `build` and will be in the path once you've sourced `devenv`.

**Comment by spacecowboy** [comment]
Yes, works then if using bash.

In zsh, `. devenv.local` will fail however. (not sure we care)

**Comment by benbc** [comment]
@spacecowboy Updated for Zsh compatibility.

File: devenv [hunk 1]
```diff
@@ -11,7 +11,7 @@ if ! which make >/dev/null; then
 fi
 
 if [[ -f devenv.local ]]; then
-    . devenv.local
+    source devenv.local
```
Comment (spacecowboy) [inline devenv]: thanks

**Pull request merged**